main/ui/src/main/java/org/cryptomator/ui/controllers/InitializeController.java

```diff
@@ -100,6 +124,49 @@ public void setListener(InitializationListener listener) {
 		this.listener = Optional.ofNullable(listener);
 	}
 
+	/* Methods */
+
+	private void checkPasswordStrength(String password) {
```

Comment:
This is in my opinion too much copy/paste code. One way to solve it might be to use an `Binding<Integer> passwordStrength` which is calculated by zxcvbn. Then bind the shapes color and width as well as the labels textProperty to passwordStrength using `EasyBind.map(...)`.

The functions used for the mapping can then be externalized as static methods to some utility.